docs/usage/modules/manager.md

```diff
@@ -85,16 +87,17 @@ To disable all managers within a language like `python`, do this:
 }
 ```
 
-Note: Only languages declared by a Renovate manager can be supported, so please verify first.
+Only languages declared by a Renovate manager can be supported, so please verify first.
```

Comment:
```markdown
Only languages declared by a Renovate manager can be supported, so please verify first.
```

Where should a reader check what languages are supported?

Reply:
see https://docs.renovatebot.com/modules/manager/#supported-managers
 on the left are the `languages`, `other` means no language

Reply:
Thanks @viceice, I've added a link to that section in the document. :smile: